templates/hooks/serviceAccount.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "betydb.fullname" . }}-hooks
```

Comment:
do we need a separate account for the hooks? Not sure we need all the permissions listed below for the hooks.

Reply:
using separate service account as permissions required for bety and bety hooks are different.
Will remove endpoint, service access from bety hook role.